docs/userguide/dockervolumes.md

```diff
@@ -216,7 +216,8 @@ then mounting the volumes from the `dbstore` container hides the
 from the `dbstore` container are visible.
 
 You can use multiple `--volumes-from` parameters to bring together multiple data
-volumes from multiple containers.
+volumes from multiple containers. More details about `--volumes-from` can be check
```

Comment:
This one; https://github.com/docker/docker/pull/17926#discussion_r45570877 (hope that link works, GitHub can be a bit picky)